### PR TITLE
feat: Add helper to repeat code blocks

### PR DESCRIPTION
**Pull request by furzeface**
I find this helper pretty useful. Can be useful to avoid repetition repetition. See styleguide for example.

### CODE REVIEW

**Comment by tawashley** [comment]
@furzeface Ha, that's awesome!

File: views/styleguide.hbs [hunk 1]
```diff
@@ -21,41 +21,29 @@
...
+					{{#repeat 6}}
 						{{>grid/grid gridRef='col--1/6' gridItemClass='styleguide__grid-item' content='col--1/6'}}
-						{{>grid/grid gridRef='col--1/6' gridItemClass='styleguide__grid-item' content='col--1/6'}}
-						{{>grid/grid gridRef='col--1/6' gridItemClass='styleguide__grid-item' content='col--1/6'}}
-						{{>grid/grid gridRef='col--1/6' gridItemClass='styleguide__grid-item' content='col--1/6'}}
-						{{>grid/grid gridRef='col--1/6' gridItemClass='styleguide__grid-item' content='col--1/6'}}
-						{{>grid/grid gridRef='col--1/6' gridItemClass='styleguide__grid-item' content='col--1/6'}}
+					{{/repeat}}
 					</div>
+					{{#repeat 4}}
```
Comment (bmds) [inline views/styleguide.hbs]: The repeat block is the wrong side of the div here

File: views/styleguide.hbs [hunk 2]
```diff
@@ -21,41 +21,29 @@
...
+					{{/repeat}}
 					</div>
+					{{#repeat 3}}
 					<div class="grid__row">
 						{{>grid/grid gridRef='col--1/3' gridItemClass='styleguide__grid-item' content='col--1/3'}}
-						{{>grid/grid gridRef='col--1/3' gridItemClass='styleguide__grid-item' content='col--1/3'}}
-						{{>grid/grid gridRef='col--1/3' gridItemClass='styleguide__grid-item' content='col--1/3'}}
+					{{/repeat}}
 					</div>
+					{{#repeat 2}}
```
Comment (bmds) [inline views/styleguide.hbs]: And here

**Comment by bmds** [comment]
That's really useful, nice work

**Comment by bmds** [comment]
@furzeface Tried to assign both of us there? Annoying isn't it

**Comment by furzeface** [comment]
Haha my bad, sorry. Nice one :)

**Pull request merged**